src/wallet.ts

```diff
@@ -1204,32 +1209,32 @@ export class Wallet {
         STORE_CONTRACT_CALL_METHODS,
     })
 
-    const _royalties = token.royaltys.reduce(
+    const _royalties = royaltys ? royaltys.reduce(
       (accumulator, { account, percent }) => {
         return {
           ...accumulator,
           [account]: percent,
         }
       },
       {}
-    )
+    ) : null;
     // @ts-ignore: method does not exist on Contract type
     const { base_uri } = await contract.nft_metadata()
 
+    const royaltiesObj = _royalties ?  Object.keys(_royalties).length > 0
+    ? {
+        split_between: _royalties,
+        percentage: token.royaltyPercent,
+      } : null : null;
```

Comment:
Same lint question about nested ternaries, thought Airbnb had a rule against them, I do find them a bit harder to read